code/modules/mob/living/living.dm

```diff
@@ -572,8 +573,8 @@ Sorry Giacom. Please don't be mad :(
 		if(get_dist(src, pulling) > 1 || ((pull_dir - 1) & pull_dir)) //puller and pullee more than one tile away or in diagonal position
 			if(isliving(pulling))
 				var/mob/living/M = pulling
-				if(M.lying && !M.buckled && (prob(M.getBruteLoss() / 2)))
-					makeTrail(T, M)
+				if(M.lying && !M.buckled && (prob(M.getBruteLoss()*200/M.maxHealth)))
```

Comment:
we need to use the ratio of brute damage to maxhealth because of living mobs with very low health.